rotkehlchen/tests/unit/decoders/test_zerox.py

```diff
@@ -1232,3 +1232,381 @@ def test_swap_anon_event(ethereum_inquirer: 'EthereumInquirer', ethereum_account
         counterparty=CPT_COWSWAP,
         address=address,
     )]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize(
+    'ethereum_accounts',
+    [[
+        '0x8722bfc146EE513cfE97f1ca52343fbA63Cc718a',
```

Comment:
What are you doing here? This is not how we test stuff.

In a swap you should track `1` address. Not 6. 

All the tests here do not correspond to how we test things.

Reply:
I updated the tests to tests the swaps using the version 1.7 of settler contract.